### ISSUE TITLE
Std. dev of per channel L2 breakdown in CLI seems... wrong

### ISSUE THREAD

**Issue by arghdos** [AMDResearch/omniperf#134]
See: 

![image](https://github.com/AMDResearch/omniperf/assets/6463881/b5f6f4cf-7468-4f36-b67f-0ed6dcb8ee5d)

vs:

![image](https://github.com/AMDResearch/omniperf/assets/6463881/22479eba-f973-4e74-ae7f-8f9ae5e935cc)
![image](https://github.com/AMDResearch/omniperf/assets/6463881/5beedfb5-1d44-4f64-a264-202402ca582a)

It's extremely hard to follow what's going on in this autogenerated section here:

https://github.com/AMDResearch/omniperf/blob/8af6efc0be5288dadd2e758556154e52f7406e5d/src/omniperf_analyze/configs/gfx90a/1800_L2_cache_per_channel.yaml#L60

**Comment by feizheng10** [comment]
Ping me...I might provide something different:)
